app/models/actual.rb

```diff
@@ -1,4 +1,12 @@
 class Actual < Transaction
+  has_one :comment,
+    -> { where(commentable_type: "Actual") },
+    foreign_key: :commentable_id,
+    dependent: :destroy,
+    autosave: true
+
+  # validates_associated :comment
```

Comment:
We should either remove this comment or un-comment it. Was this a temporary measure during development?

Reply:
Removing comment, rerunning tests.